#! /bin/sh /usr/share/dpatch/dpatch-run
## 01_SConstruct_debian.dpatch by Florian Ernst <florian@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Modify SConstruct for the Debian environment

@DPATCH@
diff -urNad blender-2.41~/SConstruct blender-2.41/SConstruct
--- blender-2.41~/SConstruct	2006-01-26 11:16:31.000000000 +0100
+++ blender-2.41/SConstruct	2006-01-26 11:40:08.000000000 +0100
@@ -70,7 +70,7 @@
 if sys.platform == 'linux2' or sys.platform == 'linux-i386':
 	use_international = 'true'
 	use_gameengine = 'true'
-	use_openal = 'true'
+	use_openal = 'false'
 	use_fmod = 'false'
 	use_quicktime = 'false'
 	use_sumo = 'true'
@@ -80,8 +80,8 @@
 	use_fluidsim = 'true'
 	build_blender_dynamic = 'true'
 	build_blender_static = 'false'
-	build_blender_player = 'false'
-	build_blender_plugin = 'false'
+	build_blender_player = 'true'
+	build_blender_plugin = 'true'
 	release_flags = ['-O2']
 	debug_flags = ['-O2', '-g']
 	extra_flags = ['-pipe', '-funsigned-char']
@@ -141,16 +141,16 @@
 	python_include = [sysconfig.get_python_inc ()]
 	python_linkflags = Split (sysconfig.get_config_var('LINKFORSHARED'))
 	# International support information
-	ftgl_lib = []
-	ftgl_libpath = []
-	ftgl_include = ['#extern/bFTGL/include']
+	ftgl_lib = ['ftgl']
+	ftgl_libpath = ['/usr/lib']
+	ftgl_include = ['/usr/include/FTGL']
 	freetype_env.ParseConfig ('pkg-config --cflags --libs freetype2 2>/dev/null || freetype-config --cflags --libs 2>/dev/null')
 	freetype_lib = freetype_env.Dictionary()['LIBS']
 	freetype_libpath = freetype_env.Dictionary()['LIBPATH']
 	freetype_include = freetype_env.Dictionary()['CPPPATH']
-	gettext_lib = []
-	gettext_libpath = []
-	gettext_include = []
+	gettext_lib = ['gettextpo']
+	gettext_libpath = ['/usr/lib']
+	gettext_include = ['/usr/include']
 	# OpenAL library information
 	openal_lib = ['openal']
 	openal_libpath = ['/usr/lib']
